### PR TITLE
fixed itruediv tests

### LINKED ISSUE #11668: Fix ndarray.test_numpy_instance_itruediv__

### PR DESCRIPTION
**Pull request by rarisenpai**
Close #11668

### CODE REVIEW

**Comment by Infrared1029** [comment]
hi @rarisenpai!, PR looks good, just a tiny little comment and we are good to merge!

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py [hunk 1]
```diff
@@ -2085,7 +2085,7 @@ def test_numpy_instance_itruediv__(
     frontend,
 ):
     input_dtypes, xs = dtype_and_x
-
+    assume(not np.any(np.isclose(xs[1], 0.0)))
```
Comment (Infrared1029) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py:2088]: this is probably unnecessary, as division by 0 results in `inf` which can be handled, we can remove this line.

Comment (rarisenpai) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py:2088]: okay let me get to that

**Comment by rarisenpai** [comment]
hi I was resolving the conflicts between files done now

**Comment by Infrared1029** [comment]
There appear to be some linitng issues flake8 is complaining about, can you fix those as well? installed pre-commit will help with this a lot.

**Comment by rarisenpai** [comment]
I took a look at the logs and those flake8 errors aren't mine as I had run precommit

**Comment by rarisenpai** [comment]
I took a look at the logs and those flake8 errors aren't mine as I had run precommit

**Comment by Infrared1029** [comment]
lgtm!

**Pull request merged**